pkg/codegen/python/gen.go

```diff
@@ -1199,14 +1185,32 @@ func GeneratePackage(tool string, pkg *schema.Package, extraFiles map[string][]b
 		_ = getMod(":config/config:")
 	}
 
-	scanResource := func(r *schema.Resource) {
+	scanResource := func(r *schema.Resource) error {
 		mod := getMod(r.Token)
 		mod.resources = append(mod.resources, r)
+
+		// Calculate casing tables. We do this up front because our docstring generator (which is run during
+		// genResource) requires them.
```

Comment:
This comment doesn't appear to be correct anymore - and does the comment point to any potential issue with this change?

Reply:
> This comment doesn't appear to be correct anymore

The comment is still correct. Which part seems wrong?

>  - and does the comment point to any potential issue with this change?

It pointed to the issue that these changes are fixing.